web-app/src/components/jobs/job-details/job-share-modal.tsx

```diff
@@ -341,19 +355,11 @@ export default function JobShareModal({
                     {t("publicAccessDescription")}
                   </p>
                 </div>
-                {jobShare?.accessType === ShareAccessType.PUBLIC && (
+                {jobShare?.isPublic && (
                   <Check className="text-semantic-success size-4" />
                 )}
               </div>
-              {activeOrganizationId && !organization ? (
-                <div className="flex items-center gap-2 p-4">
-                  <Skeleton className="size-6 rounded-full" />
-                  <div className="flex-1 space-y-2">
-                    <Skeleton className="h-4 w-32" />
-                    <Skeleton className="h-3 w-48" />
-                  </div>
-                </div>
-              ) : organization ? (
+              {activeOrganizationId && organization ? (
```

Comment:
<img width="617" height="566" alt="image" src="https://github.com/user-attachments/assets/8c4ad5c4-ee2d-414a-9c38-811baabd6669" />

When we're on the Personal scope, it keeps loading this Org section, it didn't stop loading at all.

Reply:
Good catch, thanks. It's fixed with the commit 2203a0498309dc2a15d4a4b8ac3f18c0811930a1.